playground/main.ts

```diff
@@ -33,7 +33,7 @@ declare global {
     // added in this file
     getWidgetInstance: () => OktaSignInAPI;
     createWidgetInstance: (options: WidgetOptions) => OktaSignInAPI;
-    renderPlaygroundWidget: (options: WidgetOptions) => void;
+    renderPlaygroundWidget: (options: WidgetOptions & { assertNoEnglishLeaks?: boolean, customize?: boolean }) => Promise<void>;
```

Comment:
Returning the `Promise` is an addition, the union type in the parameter is just fixing the previous type to match reality.